Shared/Samples/List spatial reference transformations/ListSpatialReferenceTransformationsView.swift

```diff
@@ -46,16 +46,19 @@ struct ListSpatialReferenceTransformationsView: View {
                 }
                 .errorAlert(presentingError: $error)
             
-            NavigationStack {
-                TransformationsList(model: model)
-                    .navigationTitle("Transformations")
-                    .navigationBarTitleDisplayMode(.inline)
-                    .toolbar {
-                        ToolbarItem(placement: .topBarTrailing) {
-                            transformationsMenu
-                        }
-                    }
+            ZStack {
+                Text("Transformations")
+                    .bold()
+                HStack {
+                    Spacer()
+                    transformationsMenu
+                        .labelStyle(.iconOnly)
+                }
+                .padding()
             }
+            .background(Color(.systemGray6))
```

Comment:
did you try this on both dark and light mode?

Reply:
No, good catch! It should work as expected now.